RubeesEat/appsettings.json

```diff
@@ -1,9 +1,12 @@
-{
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft.AspNetCore": "Warning"
-    }
-  },
-  "AllowedHosts": "*"
-}
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "ConnectionStrings": {
+    "DefaultConnectionString": "Server=localhost;Database=RubeesEatDB;UID=root;PW=;"
```

Comment:
We don't use this configuration property anymore? We might want to consider using this configuration property instead of manually doing an `environement.getEnvironementVariable` as IConfiguration already supports overriding values using environment variables.